### DISCUSSION TITLE
Export products with blocks

### DISCUSSION THREAD

**Discussion by julescournut** [pimcore/pimcore#16204] [Q&A / Support]
Hello, 
I'm on Pimcore v.11.1.0 and my products have [blocks fields](https://pimcore.com/docs/platform/Pimcore/Objects/Object_Classes/Data_Types/Blocks/). I want to export my products data so I tried to go in grid view and add my block field to the grid options but it says that it is not supported.
![Capture d’écran de 2023-11-06 10-41-48](https://github.com/pimcore/pimcore/assets/26310511/96026952-6dc2-427c-ac7a-a8c2750c4db5)

Is there any other way to export those data ?

**Comment by Maksonis** [comment]
Hi, you can use Operator PHP Code (https://pimcore.com/docs/pimcore/10.6/User_Documentation/DataObjects/Grid_Configuration_Operators/Others/PHPCode.html) in grid and write code to extract block data. Another way is to use Any Getter (https://pimcore.com/docs/pimcore/10.6/User_Documentation/DataObjects/Grid_Configuration_Operators/Extractors/AnyGetter.html) in combination with custom method e.g. getBlockData().
